src/components/utils/test/thread_test.cc

```diff
@@ -74,14 +78,18 @@ class ThreadTestDelegate : public threads::ThreadDelegate {
   void WaitForThreadStart() {
     AutoLock test_lock(test_mutex_);
     while (!thread_started_) {
-      cond_var_.Wait(test_lock);
+      cond_var_.WaitFor(test_lock, kMilliSecondsToWait);
```

Comment:
@wolfylambova22 Please, change the cond_var_ name to something more descriptive. What is it for? What's the purpose?